app/views/tag/show.html.erb

```diff
@@ -13,7 +13,11 @@
     <small>parent: <%= @tags.first.parent %></small>
   <% end %>
 
-  <%= render :partial => "notes/format_toggle" if @node_type == "note" %>
+  <div class="btn-group visible-desktop" style="float:right;" id="note-format">
```

Comment:
Hi, this is awesome! I do notice this file changed, though, which seems to be related to a separate issue. Would you mind fetching a prior version of this file and re-submitting without this particular set of modifications? Thank you!!!

Reply:
Hi, thanks!
Just committed with unrelated issue changed back! Hope it's the same as to fetch initial version.